crypto/bn/bn_rsa_fips186_4.c

```diff
@@ -48,6 +48,38 @@ const BIGNUM ossl_bn_inv_sqrt_2 = {
     BN_FLG_STATIC_DATA
 };
 
+/*
+ * Refer to FIPS 186-5 Table B.2 for minimum rounds of Miller Rabin
+ * required for generation of RSA aux primes (p1, p2, q1 and q2)
+ * using an error probability of 2^–100.
+ */
+static int bn_rsa_fips186_5_aux_prime_MR_rounds(int nbits)
+{
+    if (nbits >= 4096)
+        return 22;
+    if (nbits >= 3072)
+        return 27;
+    if (nbits >= 2048)
+        return 32;
+    return 0;
```

Comment:
I'm not sure a default of 0 makes sense.

Reply:
It indicates an error condition. It never uses this value here, as the code that calls it checks the error returned from the aux_prime_min_size().